pravega-client-examples/README.md

```diff
@@ -152,7 +153,7 @@ First, ensure that the Pravega cluster that the applications are going to work w
 Auth (authentication and authorization) enabled. See Pravega documentation for how to enable TLS and Auth.
 
 Now, you might want to run `SecureWriter` in one window and `SecureReader` in another window.
-
+and finally `SecureBatchReader`
```

Comment:
This looks disconnected from the preceding sentence, right? Maybe the dot should be a comma.